cmd/syncer/main.go

```diff
@@ -0,0 +1,196 @@
+/*
+Copyright 2022 The MultiCluster Traffic Controller Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/sirupsen/logrus"
```

Comment:
why logrus?
`sigs.k8s.io/controller-runtime/pkg/log` is used elsewhere

Reply:
Looks like it's using go-lgr, so we could import that directly if you didn't want to use controller-runtime here?
```
All logging in controller-runtime is structured, using a set of interfaces defined by a package called logr (https://pkg.go.dev/github.com/go-logr/logr)
```

Reply:
I didn't think too hard about this, honestly. Happy to change it.